examples/gemm/sgemm.py

```diff
@@ -38,7 +38,7 @@ def sgemm(A, B,
     code = read_code(sgemm_file, params=config)
     kern = load_kernel('sgemm', code)
 
-    grid = (math.ceil(m / blk_m), math.ceil(n / blk_n), 1)
+    grid = (int(math.ceil(m / blk_m)), int(math.ceil(n / blk_n)), 1)
```

Comment:
Wouldn't it change the behavior?
For instance, when `m=3` and `blk_m=2`, `math.ceil(m / blk_m) == 2` and `m // blk_m == 1`.